glossary.md

```diff
@@ -13,20 +13,21 @@ Nếu một từ chưa có trong bảng thuật ngữ dưới đây, các bạn
 
 ## Anh - Việt
 
+
 | English                           | Tiếng Việt (Nếu bạn bổ sung thì KHÔNG căn chỉnh các dòng khác) | Thảo luận tại                |
 |-----------------------------------|----------------------------------------------------------------|------------------------------|
 | accuracy                          | độ chính xác                                                   |                              |
-| benchmark                         |                                                                | [#87](http://bit.ly/2BvfPYA) |
+| benchmark                         | đánh giá xếp hạng                                              | [#87](http://bit.ly/2BvfPYA) |
```

Comment:
- benchmark theo em hiểu, nó là biểu đồ được tính toán đo lường để biểu diễn (vẽ) ra theo một độ đo nào đó bằng biểu đồ (thường là biểu đồ đường: độ đo / thời gian) để người ta có thể biểu thị được chất lượng hoạt động của một tính năng nào đó. Còn dùng cho để đánh giá, xếp hạng hay thống kê gì thì tùy vào mục đích của người vẽ nên. Nên em nghĩ là dịch là `đánh giá xếp hạng` không được chính xác lắm. Em nghĩ nên để tiếng anh luôn.

Reply:
Benchmark không phải là biểu đồ tính toán. Biểu đồ tính toán chỉ là một cách họ thể hiện benchmark - đánh giá xếp hạng thôi em.